### PR TITLE
Spanish Translations

### PR DESCRIPTION
**Pull request by akkys77**
My PR is a:

💅 Enhancement

Main update on the:
Admin

Provide spanish translations for admin panel and plugins.
I create the following files using the local en.json as a reference
./packages/strapi-plugin-content-manager/admin/src/translations/es.json
./packages/strapi-plugin-users-permissions/admin/src/translations/es.json
./packages/strapi-plugin-content-type-builder/admin/src/translations/es.json
./packages/strapi-plugin-email/admin/src/translations/es.json
./packages/strapi-plugin-upload/admin/src/translations/es.json
./packages/strapi-admin/admin/src/translations/es.json
./packages/strapi-plugin-settings-manager/admin/src/translations/es.json
./packages/strapi-generate-plugin/files/admin/src/translations/es.json

### CODE REVIEW

File: packages/strapi-admin/admin/src/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  "app.components.Button.save": "Guardar",
+  "app.components.Button.cancel": "Cancelar",
+
+  "app.components.ComingSoonPage.comingSoon": "Próximamente",
+  "app.components.ComingSoonPage.featuresNotAvailable": "Esta característica está aún en desarrollo.",
+
+  "app.components.DownloadInfo.download": "Descarga en curso...",
+  "app.components.DownloadInfo.text": "Esto puede tardar un minuto. Gracias por su paciencia.",
+
+  "app.components.HomePage.welcome": "Bienvenidos a bordo!",
```
Comment (maturano) [inline packages/strapi-admin/admin/src/translations/es.json]: Missing open exclamation point

File: packages/strapi-admin/admin/src/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  "app.components.HomePage.welcome.again": "Bienvenido ",
+  "app.components.HomePage.cta": "CONFIRMAR",
+  "app.components.HomePage.community": "Encuentra la comunidad en la web",
+  "app.components.HomePage.newsLetter": "Suscríbase al boletín de noticias para ponerse en contacto con Strapi",
+  "app.components.HomePage.community.content": "Discutir con los miembros del equipo, colaboradores y desarrolladores en diferentes canales.",
+  "app.components.HomePage.create": "Crea tu primer Tipo de Contenido",
+  "app.components.HomePage.welcomeBlock.content": "Estamos felices de tenerlo como miembro de la comunidad. Estamos constantemente en busca de comentarios así que no dude en enviarnos DM en\u0020",
```
Comment (maturano) [inline packages/strapi-admin/admin/src/translations/es.json]: Isn't `\u0020` just an space?; https://www.fileformat.info/info/unicode/char/0020/index.htm

File: packages/strapi-admin/admin/src/config/languages.json [hunk 3]
```diff
@@ -1,3 +1,16 @@
 {
-  "languages": ["ar", "en", "fr", "de", "it", "pl", "pt", "pt-BR", "tr", "zh", "zh-Hans"]
+  "languages": [
```
Comment (maturano) [inline packages/strapi-admin/admin/src/config/languages.json]: IMHO, is better the one line syntax

File: packages/strapi-admin/admin/src/translations/es.json [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  "app.components.HomePage.welcomeBlock.content.again": "Esperamos que estés progresando en tu proyecto.... Siéntase libre de leer las últimas novedades sobre Strapi. Estamos dando lo mejor de nosotros mismos para mejorar el producto basándonos en sus comentarios.",
+  "app.components.HomePage.welcomeBlock.content.issues": "temas.",
+  "app.components.HomePage.welcomeBlock.content.raise": "\u0020o plantear\u0020",
+  "app.components.HomePage.createBlock.content.first": "El\u0020",
+  "app.components.HomePage.createBlock.content.second": "\u0020le ayudará a definir la estructura de datos de sus modelos. Si eres nuevo aquí, te recomendamos encarecidamente que sigas nuestros\u0020",
+  "app.components.HomePage.createBlock.content.tutorial": "\u0020tutorial.",
+  "app.components.HomePage.button.quickStart": "INICIAR EL TUTORIAL DE INICIO RÁPIDO",
+  "app.components.HomePage.button.blog": "VER MÁS EN EL BLOG",
+  "app.components.HomePage.support": "APOYANOS",
```
Comment (maturano) [inline packages/strapi-admin/admin/src/translations/es.json]: Tilde en la O: APÓYANOS

File: packages/strapi-admin/admin/src/translations/es.json [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+  "app.components.HomePage.support.content": "¡Al comprar la camiseta, nos permitirá continuar nuestro trabajo en el proyecto para darle la mejor experiencia posible!",
+  "app.components.HomePage.support.link": "CONSIGUE TU CAMISETA AHORA",
+
+  "app.components.BlockLink.documentation": "Lea la documentación",
+  "app.components.BlockLink.documentation.content": "Descubra los conceptos, guías de referencia y tutoriales.",
+  "app.components.BlockLink.code": "Ejemplos de código",
+  "app.components.BlockLink.code.content": "Aprenda probando proyectos reales desarrollados por la comunidad.",
+
+
```
Comment (maturano) [inline packages/strapi-admin/admin/src/translations/es.json]: Are this double spaces in the original? 🤔

File: packages/strapi-admin/admin/src/translations/es.json [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+  "app.components.InstallPluginPopup.navLink.screenshots": "Capturas de pantalla",
+  "app.components.InstallPluginPopup.navLink.avis": "opinión",
+  "app.components.InstallPluginPopup.navLink.faq": "preguntas frecuentes",
+  "app.components.InstallPluginPopup.navLink.changelog": "changelog",
+  "app.components.InstallPluginPopup.noDescription": "No hay descripción disponible",
+
+  "app.components.LeftMenuFooter.poweredBy": "Potenciado por ",
+  "app.components.LeftMenuLinkContainer.configuration": "Configuraciones",
+  "app.components.LeftMenuLinkContainer.general": "General",
+  "app.components.LeftMenuLinkContainer.installNewPlugin": "Plaza de mercado",
```
Comment (maturano) [inline packages/strapi-admin/admin/src/translations/es.json]: Not sure about this 🤔 ... I don't know where is used, but without see it I think is better just "Mercado"

File: packages/strapi-plugin-users-permissions/admin/src/translations/es.json [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+  "Email.template.reset_password": "Restablecer contraseña",
+  "Email.template.success_register": "Inscripción exitosa",
+
+  "Auth.advanced.allow_register": "",
+
+  "PopUpForm.button.cancel": "Cancelar",
+  "PopUpForm.button.save": "Guardar",
+  "PopUpForm.header.add.providers": "Añadir nuevo proveedor",
+  "PopUpForm.header.edit.email-templates": "Editar Plantillas de Email",
+  "PopUpForm.header.edit.providers": "Editar {provider} Proveedor",
```
Comment (maturano) [inline packages/strapi-plugin-users-permissions/admin/src/translations/es.json]: 🤔 Again, without see where is used, I think could be better "Editar Proveedor {provider}"

File: packages/strapi-plugin-users-permissions/admin/src/translations/es.json [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+  "notification.success.delete": "El elemento ha sido borrado",
+  "notification.success.submit": "Los ajustes se han actualizado",
+
+  "plugin.description.short": "Proteja su API con un proceso de autenticación completo basado en JWT",
+  "plugin.description.long": "Proteja su API con un proceso de autenticación completo basado en JWT. Este plugin viene también con una estrategia ACL que le permite administrar los permisos entre los grupos de usuarios.",
+
+  "Plugin.permissions.application.description": "Defina todas las acciones permitidas de su proyecto.",
+  "Plugin.permissions.plugins.description": "Defina todas las acciones permitidas para el plugin {name}.",
+
+  "Plugins.header.title": "Permisiones",
```
Comment (maturano) [inline packages/strapi-plugin-users-permissions/admin/src/translations/es.json]: "Permisos"? 🤔

File: packages/strapi-plugin-users-permissions/admin/src/translations/es.json [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+
+  "components.Input.error.password.noMatch": "Las contraseñas no coinciden",
+
+  "Controller.input.label": "{label} ",
+  "Controller.selectAll": "Seleccionar todo",
+
+  "EditForm.inputSelect.label.role": "Rol predeterminado para usuarios autenticados",
+  "EditForm.inputSelect.description.role": "Adjuntará el nuevo usuario autenticado al rol seleccionado.",
+  "EditForm.inputSelect.subscriptions.label": "Gestionar cuotas de suscripción",
+  "EditForm.inputSelect.subscriptions.description": "Limite el número de suscripciones por IP por hora.",
```
Comment (maturano) [inline packages/strapi-plugin-users-permissions/admin/src/translations/es.json]: "suscripciones *de* IP por hora"??? 🤔

File: packages/strapi-plugin-users-permissions/admin/src/translations/es.json [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+  "Auth.form.register.email.label": "Email",
+  "Auth.form.register.email.placeholder": "johndoe@gmail.com",
+  "Auth.form.register.username.label": "Nombre de usuario",
+  "Auth.form.register.username.placeholder": "John Doe",
+  "Auth.form.register.password.label": "Contraseña",
+  "Auth.form.register.confirmPassword.label": "Contraseña de confirmación",
+  "Auth.form.register.news.label": "Manténgame informado sobre las nuevas características y las próximas mejoras.",
+
+  "Auth.form.register-success.email.label": "Email enviado con éxito a",
+  "Auth.form.register-success.email.placeholder": "mysuperemail@gmail.com",
```
Comment (maturano) [inline packages/strapi-plugin-users-permissions/admin/src/translations/es.json]: :) is that way in the original? ...

File: packages/strapi-plugin-users-permissions/admin/src/translations/es.json [hunk 11]
```diff
@@ -0,0 +1,175 @@
+{
+  "Auth.form.button.register-success": "Enviar de nuevo",
```
Comment (maturano) [inline packages/strapi-plugin-users-permissions/admin/src/translations/es.json]: Alternative: "Enviar nuevamente"

**Comment by lauriejim** [comment]
@akkys77 I let you discuss with @maturano about his feedback. I'm not 🇪🇸 expert.
So let me know when this PR is ready to merge.

**Comment by maturano** [comment]
🤔 ... I would not like this get lost. @lauriejim I don't see problem with merge this PR, there is nothing wrong.

Once merged I promise review it in context and make the corrections if necessary.

**Comment by lauriejim** [comment]
@maturano okay good ! So let's merge it.

**Pull request merged**